lib/connections/service.go

```diff
@@ -641,3 +646,30 @@ func tlsTimedHandshake(tc *tls.Conn) error {
 	defer tc.SetDeadline(time.Time{})
 	return tc.Handshake()
 }
+
+func IsAllowedNetwork(host string, allowed []string) bool {
+	if len(allowed) == 0 {
```

Comment:
Perhaps this should be on the deviceCfg struct, as the special case of empty list makes it somewhat a shit generic function.